hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/AbiConstants.java

```diff
@@ -131,20 +133,38 @@ private AbiConstants() {
     // **** HIP-358 function selectors ****
     // createFungibleToken(HederaToken memory token, uint initialTotalSupply, uint decimals)
     public static final int ABI_ID_CREATE_FUNGIBLE_TOKEN = 0x7812a04b;
+    // createFungibleToken(HederaToken memory token, uint64 initialTotalSupply, uint32 decimals)
+    public static final int ABI_ID_CREATE_FUNGIBLE_TOKEN_V2 = 0xc23baeb6;
```

Comment:
nit: I like the callout comment you did on the others so suggesting this
```suggestion
    // decimals updated to uint32
    public static final int ABI_ID_CREATE_FUNGIBLE_TOKEN_V2 = 0xc23baeb6;
```